app/presenters/idv/jurisdiction_failure_presenter.rb

```diff
@@ -30,7 +30,7 @@ def description
     end
 
     def message
-      t('headings.lock_failure')
+      nil
```

Comment:
What is going on here? Why did we remove this translation?

Reply:
This is the heading that says "Here is what you can do next". What follows for both direct login and SP users is a link that either takes them to the help link for the SP or back to the account page. Given wording of the heading, it sounds like the user will be given a list of steps, not a link "Back to .."